src/js/lib/monster.ui.js

```diff
@@ -3844,18 +3844,16 @@ define(function(require) {
 			throw TypeError('"options" is not a plain object');
 		}
 
-		var size = args.size,
+		var size = _.get(args, 'size', 0),
```

Comment:
Consequently, `args.size` and `args` are now optional, please reflect that in the jsdoc and [doc ref](https://github.com/2600hz/monster-ui/blob/master/docs/monster-ui/charsRemaining().md).